sdk/keyvault/keyvault-certificates/test/public/utils/common.ts

```diff
@@ -4,7 +4,12 @@
 import { env } from "@azure-tools/test-recorder";
 import { assert } from "chai";
 import { SupportedVersions, supports, TestFunctionWrapper } from "@azure/test-utils";
-import { LATEST_API_VERSION, CertificateClientOptions } from "../../src/certificatesModels";
+import { CertificateClientOptions } from "../../../src";
+
+/**
+ * The latest supported KeyVault service API version
+ */
+export const LATEST_API_VERSION = "7.3-preview";
```

Comment:
Can we infer this from https://github.com/Azure/azure-sdk-for-js/pull/19959/files#diff-8182aee8eae78191aaa9301924729aafe2786e1f58fd5bd5bc31af8c28405d23R43 instead of redefining it? I just worry about having yet another place to change you know?